### PR TITLE
adjust db get-item schema

### PR DESCRIPTION
**Pull request by osahyoun**
See docs here: https://docs.aws.amazon.com/sdk-for-ruby/v3/api/Aws/DynamoDB/Table.html#get_item-instance_method

```ruby
table.get_item({
  key: { # required
    "AttributeName" => "value", # value <Hash,Array,String,Numeric,Boolean,IO,Set,nil>
  },
  attributes_to_get: ["AttributeName"],
  consistent_read: false,
  return_consumed_capacity: "INDEXES", # accepts INDEXES, TOTAL, NONE
  projection_expression: "ProjectionExpression",
  expression_attribute_names: {
    "ExpressionAttributeNameVariable" => "AttributeName",
  },
})
```

### CODE REVIEW

**Review by yeseniamolinab — APPROVED** [review]
@osahyoun This is how it was before what I did on #208, which has been constantly failing (before my change).
I am not opposed to this change, as the one I made didn't work, but since this is the same as before, it won't fix the issue either, right?

**Comment by osahyoun** [comment]
@yeseniamolinab how funny - you must think me stupid! I really should pay more attention to what came before. So, this original schema works fine when executed locally, so I'm not sure what's going on in production. I test now if the same error is raised when @postcode is null.

File: app/services/get_constituency.rb [hunk 1]
```diff
@@ -11,6 +11,7 @@ def initialize(postcode, client)
   end
 
   def get
+    return nil if @postcode.nil? || @postcode.empty?
```
Comment (osahyoun) [inline app/services/get_constituency.rb:14]: @yeseniamolinab nil/empty postcode was the issue after all. Merging.

Comment (yeseniamolinab) [reply app/services/get_constituency.rb:14]: Thanks, Omar!

**Pull request merged**